ext-src/mscclpp_ibv_access_relaxed_ordering.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index a95a8e5..be69659 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -96,6 +96,28 @@ include(${PROJECT_SOURCE_DIR}/cmake/AddFormatTargets.cmake)
+ 
+ # Find ibverbs and libnuma
+ find_package(IBVerbs)
++
++# Check if IBV_ACCESS_RELAXED_ORDERING exists in infiniband/verbs.h
++# Use internal IBVerbs definitions if it does not exist
++if(IBVERBS_FOUND)
++  try_compile(HAS_IBV_ACCESS_RELAXED_ORDERING
```

Comment:
This is a performance parameter. Disabling IB does not seem appropriate if this is not there.